sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/DefaultScheduler.java

```diff
@@ -2,7 +2,7 @@
 
 import com.google.common.annotations.VisibleForTesting;
 import com.google.protobuf.TextFormat;
-import com.mesosphere.sdk.scheduler.recovery.RecoveryPlanManagerFactory;
+import com.mesosphere.sdk.scheduler.recovery.*;
```

Comment:
IntelliJ also has a configurable limit beyond which it'll move a bunch of imports over to `*`, which I assumed was the culprit here.